css/styles.css

```diff
@@ -0,0 +1,155 @@
+/* stylelint-disable */
+* {
+    margin: 0;
+    padding: 0;
+    text-decoration: none;
+    text-align: center;
+  }
+ 
+.body h1 {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+  
+.menu {
+    display: flex;
+    gap: 30px;
+    justify-content: center;
+    list-style: none;
+    margin: 20px;
+  }
+  
+  .h-menu {
+    display: none;
+  }
+  
+.body h2 {
+    color: #30a02c;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 10px;
+  }
+  
+.body p {
+    color: #172b4d;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    margin-top: 40px;
+  }
+  
+.icon {
+    margin-top: 40px;
+  }
+  
+.icon ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+  }
+  
+.icon a {
+    margin: 10px;
+    color: #505f79;
+    font-size: 20px;
+  }
+  
+.icon a:hover {
+    cursor: pointer;
+    color: blue;
+  }
+  
+@media (max-width: 770px) {
+    .h-menu {
+      display: flex;
+      justify-content: space-between;
+      margin-top: 20px;
+      margin-right: 20px;
+      margin-left: 10px;
+      height: 60px;
+    }
+  
+    .body {
+      background-image: url(./images/header-img.png);
+      background-repeat: no-repeat;
+      background-size: cover;
+      height: 700px;
+      background-position: 100%;
+    }
+  
+    .body h1 {
+      margin-top: 60px;
+    }
+  
+    .menu {
+      display: none;
+    }
+  
+    .bar {
+      width: 30px;
+      height: 4px;
+      border-radius: 3px;
+      background: black;
+      margin: 5px;
+    }
+  
+    .body p {
+      width: 95%;
+      margin-left: 10px;
+    }
+  
+    .h-menu a {
+      font-family: "Inter", sans-serif;
+      font-style: normal;
+      font-weight: 600;
+      font-size: 20px;
+      line-height: 28px;
+      color: #28352f;
+    }
+  
+    .h-menu a:hover {
+      color: blue;
+      cursor: pointer;
+    }
+  
+    .bar:hover {
+      cursor: pointer;
+      background: blue;
+      transform: rotateX(25deg);
+    }
+  
+    .pointer li {
+      list-style-type: none;
+    }
+  
+    .pointer a:hover {
+      cursor: pointer;
+      color: rgb(190, 190, 214);
+    }
+  
+    .pointer {
+      margin-top: 80px;
+    }
+
+    .ulist {
+        display: flex;
+        list-style: none;
+        justify-content: flex-start;
```

Comment:
- [ ] Kindly make sure to make this CSS property `center` so it matches the Figma design

Your design:
![list](https://user-images.githubusercontent.com/95412943/219704599-594fca4c-3b61-4130-85c8-ba40d51794fe.PNG)
Figma design:
![list2](https://user-images.githubusercontent.com/95412943/219704643-b8eadae2-e0b9-487e-80ec-c1fcf36c870b.PNG)

Reply:
Done